cukinia

```diff
@@ -704,15 +704,18 @@ _cukinia_mount()
 
 # _cukinia_i2c: Check if i2c device exists
 # Optional: Check of a device address
+# Optional: Check the device uses the correct driver
 # arg1: the bus number
 # arg2: (optional) the device address
+# arg3: (optional) the device driver name
 _cukinia_i2c()
 {
 	local bus_number="$1"
 	local device_address="$2"
+	local driver_name="$3"
```

Comment:
could you update the doc accordingly?

Reply:
I'm not sure where I forgot to add the doc. So fat I added:

- comments to the function in cukinia script
- optional driver name argument in the README

Is there a third place I've forgotten ?

Reply:
My bad, I missed the other file